src/Symfony/Component/Json/Tests/Fixtures/templates/decode/nullable_backed_enum.string.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return static function (string $string, array $config, \Symfony\Component\Encoder\Instantiator\InstantiatorInterface $instantiator, ?\Psr\Container\ContainerInterface $services) : mixed {
+    $flags = $config['json_decode_flags'] ?? 0;
+    $providers['?Symfony\\Component\\Json\\Tests\\Fixtures\\Enum\\DummyBackedEnum'] = static function (mixed $data) : mixed {
```

Comment:
We know the more precise type of return.
```suggestion
    $providers['?Symfony\\Component\\Json\\Tests\\Fixtures\\Enum\\DummyBackedEnum'] = static function (mixed $data) :\Symfony\Component\Json\Tests\Fixtures\Enum\DummyBackedEnum {
```

If return type is not important, delete `mixed` otherwise.

Reply:
Actually, none of provider return types is really important, as it's only internal.
Do you think it is better to remove it (no matter which type it is)? And same for parameters type?

Reply:
I'm in favor of deleting them. This will prevent PHP from trying to validate types for nothing.

Reply:
Done then!